package.json

```diff
@@ -24,7 +24,9 @@
     "test": "istanbul test -- _mocha",
     "coveralls": "cat ./coverage/lcov.info | ./node_modules/.bin/coveralls",
     "apidoc": "apidoc -o apidoc-out -i src/controllers/",
-    "integration": "integration all"
+    "integration": "integration all",
+    "preversion": "npm test",
+    "postversion": "git push --follow-tags"
```

Comment:
Do you mean that `npm version` should also run `npm publish`? It does not by default